cp3-networkpolicy/egress/license-service-reporter/ibm-license-service-reporter-operator-egress.yaml

```diff
@@ -1,17 +1,15 @@
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
-  name: egress-ibm-license-service-reporter-operator
-  namespace: "lsrNamespace"
-  labels:
-    component: cpfs3
+  name: ibm-license-service-reporter-operator-kubernetes-egress
+  namespace: ibm-licensing-namespace
```

Comment:
```suggestion
  namespace: lsrNamespace
```

Reply:
please use the templated namespace name - thanks.

Reply:
done